Header header
std_msgs/String[] scene_object_name
geometry_msgs/Pose[] scene_object_pose
